hadoop-hdds/framework/src/test/java/org/apache/hadoop/hdds/server/http/TestRatisDropwizardExports.java

```diff
@@ -37,7 +37,7 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 /**
- * Test RatisDropwizardRexporter.
+ * Test RatisDropwizardExporter.
```

Comment:
Actually the class being tested is `RatisDropwizardExports`.

```suggestion
 * Test {@link RatisDropwizardExports}.
```

Reply:
done